action.yml

```diff
@@ -4,8 +4,19 @@ author: 'Netherlands eScience Center'
 inputs:
   repositories:
     required: false
-    description: 'A file contains list of GitHub repository urls'
+    description: 'A file containing list of GitHub repository urls'
     default: ''
+  curations:
+    required: false
+    description: 'A file or URL containing curations correct invalid or missing package metadata and set the concluded license for packages. See https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-curations-yml.md'
```

Comment:
```suggestion
    description: 'A file or URL containing curations correct invalid or missing package metadata and set the concluded license for packages. See https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-curations-yml.md'
    default: https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/curations.yml
```

Reply:
fixed in https://github.com/tortellini-tools/action/pull/109#discussion_r644211308, we chose to not have default curations so the arg is initialized as empty string